tests/js/setup-globals.js

```diff
@@ -64,4 +64,10 @@ global.wcSettings = {
 	},
 };
 
+const config = require( '../../config/development.json' );
+window.wcAdminFeatures = {};
+config && config.features && Object.keys( config.features ).forEach( ( key ) => {
+  window.wcAdminFeatures[ key ] = config.features[ key ];
+} );
```

Comment:
Is there any benefit on assigning each property one by one instead of assigning `config.features` to `wcAdminFeatures` directly?
```ES6
window.wcAdminFeatures = config && config.features ? config.features : {};
```

Reply:
Nope -- I've updated this line in https://github.com/woocommerce/wc-admin/pull/1500/commits/45badff93d44a9920192796be5875bab131c0a93. Thanks!